integrationTests/multiShard/hardFork/hardFork_test.go

```diff
@@ -405,6 +405,12 @@ func hardForkImport(
 			return integrationTests.MinTransactionVersion
 		}
 
+		coreArgs := componentsMock.GetCoreArgs(testscommon.GetGeneralConfig(), componentsMock.GetRunTypeCoreComponents())
+		coreArgs.NodesFilename = "../../../factory/mock/testdata/nodesSetupMock.json"
```

Comment:
This is not ok, might fail depending from where you call this func...
Can't you just pass a mock for those core comps?

Also, this "complication" of setting extra parameters to call `GetRunTypeComponents` is again related to my first comments